test-plan.md

```diff
@@ -92,6 +92,19 @@ The easist way to verify that a project was successfully loaded is to open a .cs
 * In a project that uses globbing (.NET Core), use the VS Code file explorer to add a new file next to the csproj. Intellisense/sighelp/etc should be available in the new file
 * Add a new file and reference a type in it from a different file. Deleting from disk the file containing the referenced type  should produce error messages
 
+#### Omnisharp Options
+
+  #### omnisharp.useMono (for Linux)
+  For using this option, mono version greater than or equal to 5.2.0 must be installed. If that is not so, setting this option should give an error.
+  * If the option is not set, the OmniSharp path displayed in the "OmniSharp Log" should end with "run"
+  * If the option is set, the OmniSharp path as mentioned above should end with "OmniSharp.exe"
+
+  #### omnisharp.path
```

Comment:
#### omnisharp.path
Determines which version of OmniSharp is used by the extension. Defaults to undefined.
* ##undefined## the extension uses the OmniSharp version it shipped with.
* ##Absolute Path## the extension uses a local copy of OmniSharp. The value must point to a directory which contains OmniSharp, typically a user's build output directory for the OmniSharp-Roslyn project. Example value: `C:\\omnisharp-roslyn\\artifacts\\publish\\OmniSharp.Stdio\\win7-x64\\OmniSharp.exe`.
* ##latest## the extension will download the latest build of OmniSharp produced by the OmniSharp-Roslyn repo.
* ##<version>## the extension will download a specific build of OmniSharp produced by the OmniSharp-Roslyn repo. Example value: `1.29.2-beta.60`.